config/crd/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
```

Comment:
q: are we adding `kustomize` support with this PR as well?

Reply:
Yes. The project was re-scaffolded with the new version of operator-sdk, which generated a bunch of kustomize files for us. The make deploy target in the Makefile uses kustomize to generate the yaml it applies.